bgc_data/bgc_trip.dataresource.yaml

```diff
@@ -0,0 +1,60 @@
+profile: tabular-data-resource
+name: bgc_trip
+path: https://www.cmar.csiro.au/geoserver/imos/wfs?service=wfs&version=1.1.0&request=GetFeature&outputformat=csv&typename=imos%3Abgc_trip
+title: bgc_trip
+description: A table of all BGC Trips where there are data listed
+format: csv
+mediatype: text/csv
+encoding: utf-8
+schema:
+  fields:
+  - name: PROJECTNAME
+    title: Source project name
+    type: string
+    required: true
+  - name: TRIP_CODE
+    title: Unique identifier for trip
+    type: string
+    required: true
+  - name: STATION
+    title: Station name sample was taken at
+    type: string
+    required: true
+  - name: LONGITUDE
+    title: Longitude of mid point of segment in decimal degrees East
+    type: number
+    minimum: -180
+    maximum: 180
+    required: true
+  - name: LATITUDE
+    title: Latitude of mid point of segment in decimal degrees North
+    type: number
+    required: true
+    minimum: -90
+    maximum: 90
+  - name: SAMPLEDATELOCAL
```

Comment:
I thought we agreed to code all the local to UTC times save them both being kept in all files.

Reply:
Ok, but we currently don't have any timezone info. To do the conversion, we would need to have the relevant timezone for each station in the `bgc_stationinfo` table (I think it was there at some point, but has been removed?).
Alternatively, timezones could be included in every single value of SAMPLEDATELOCAL. Pretty sure the `format` in the spec allows it to be included (though harvesting such timestamps might be a bit more error prone...).

Reply:
So the lat & long & date are normally all you need to code the UTC time, the packages know the timezones. I'm not sure how you are intending on doing this though.

Reply:
Our intention for now is to harvest all your raw tables unchanged into our (Postgresql) database and do all AODN-side processing in there, i.e. in SQL. I had a bit of a look yesterday, and couldn't find any postgresql functions that could do timezone conversions based on just lat/lon - the available functions require a specific timezone name ('Australia/Hobart', or 'AEST') as an argument.
It may be possible to do what you suggest in Python but that will require us to modify the data before ingest. I can look into that, but I think it would be easiest to just add the timezone for each station in `bgc_stationinfo`...

Reply:
I don't actually have time zones recorded, I only produce them through code. I have the state code for the NRS stations if that will that help? But then that gets more complicated for CPR. Or I can just provide local and utc time if that makes things easier. That's fairly inefficient but at least it can be automatic in the database for NRS and CPR.

Reply:
We've agreed to leave this as is for this PR, and keep track of this issue separately (#3 ).